src/components/Navbar/styledComponents.tsx

```diff
@@ -0,0 +1,98 @@
+import styled from "styled-components";
+import { minDevice } from "../../assets/styles/breakpoints";
+import { activeLink } from "../../assets/styles/colors";
+import P from "../P";
+
+const NavbarDiv = styled.div`
+  height: 5em;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 1.16em 0 0.81em;
```

Comment:
Once we have more values we may think about extracting some regularly used padding values to contants. ;)

Reply:
Ok, we can discuss it ;)